proteus/mprans/VOF3P.h

```diff
@@ -1157,9 +1159,15 @@ namespace proteus
 		      bc_u_ext = ebqe_bc_u_ext[ebNE_kb];
 		      // compute flux = flow*bc_u
 		      if (flow < 0)
-			flux_ext = bc_u_ext*flow;
-		      else 
-			flux_ext = 0;
+			{
```

Comment:
@manuel-quezada we need to be sure to set the solution at the exterior quadrature points in all of the versions of the method we want to keep using going forward. Same thing goes for NCLS3P.h